modules/main-rules/pages/data-sheets-and-characteristics.adoc

```diff
@@ -57,7 +57,11 @@ Some rules affect some unit types and not others.
 
 Speed:: The distance that you can move the unit in a single movement action (in centimetres).
 It represents a rate at which the unit can move across the battlefield and still be effective when it shoot its weapons.
-You can give units orders that allow you to move units farther than this, but their shooting will suffer.
+You can give units orders that allow you to move units farther than this, but their shooting will suffer. +
++
+Some unit data sheets show a split Speed value such as `20/30 cm`.
+Use the first value for all movement, except where the rules tell you otherwise.
```

Comment:
This is much better, but 'all movement' is too restrictive which is why I used 'all purposes'.

Reserves rolls and Distortion Cannon both reference the unit's Speed value directly, without being a form of movement.

Unless you have another word or structure that you'd prefer to use instead of 'purpose':

```suggestion
Use the first value for all purposes, except where the rules tell you otherwise.
```

Reply:
I see what you mean. One could argue that Reserves and the roll for Distortion cannon are still about movement — but it is indeed arguable and therefore ambiguous, and perhaps there are other less arguable edge cases.
I'm happy with that, I'll apply it and resolve.